### PR TITLE
Fixed typos on job page

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
@chaudum can you give this tiny update a very quick 'ack'?

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
@spanktar maybe you can take a look at this, it's a tiny change.

File: crate/pages/jobs/java-software-engineer.html [hunk 1]
```diff
@@ -1,29 +1,19 @@
-title: Java Software Engineer
-description:
-role: Software Engineering
-location: Berlin/Germany, Dornbirn/Austria
-type: Full-time
-updated: Dec 04, 2014
+title: Java Software Engineer description: role: Software Engineering location: Berlin/Germany, Dornbirn/Austria type: Full-time updated: Dec 04, 2014
```
Comment (chaudum) [inline crate/pages/jobs/java-software-engineer.html]: why did you remove newlines?
it very hard to read now :(

Comment (chaudum) [inline crate/pages/jobs/java-software-engineer.html]: also: change the `updated` date

Comment (ChrisChinchilla) [inline crate/pages/jobs/java-software-engineer.html]: That's my editor, I have a set up optimised for 'proper' markdown and it tends not to like this 'pseudo' markdown and automatically fix it. Usually I remember to correct it afterwards, sorry, will update before merging.
